File: JiHwan/programmers/table_hash_func.py
def solution(data, col, row_begin, row_end):
    answer = 0

    data = sorted(data, key=lambda x: [x[col - 1]])

    for i in range(row_begin, row_end + 1):
        total = 0

        for j in data[i - 1]:
            total += (j % i)
        answer ^= total

    return answer - 1

File: SeongYeon/js/programmers/tableHashFunc.js
function solution(data, col, row_begin, row_end) {
  data.sort((prev, next) =>
    prev[col - 1] === next[col - 1]
      ? next[0] - prev[0]
      : prev[col - 1] - next[col - 1]
  );

  const answer = data.reduce((acc, cur, idx, src) => {
    if (row_begin <= idx + 1 && row_end >= idx + 1) {
      return (
        acc ^ src[idx].reduce((acc1, cur1) => acc1 + (cur1 % (idx + 1)), 0)
      );
    }
    return acc;
  }, 0);

  return answer;
}
